specification.md

```diff
@@ -2508,10 +2508,25 @@ to the trigger/produced event.
 
 #### SubFlowRef Definition
 
+`subFlowRef` definition can have two types, either `string` or `object`.
+If `string` type, it defines the unique id of the sub-workflow to be invoked. 
+This short-hand definition can be used if sub-workflow lookup is done only by its `id`
+property and not it's `version` property and if the default value of `waitForCompletion` is assumed.
+
+```json
+{
+  "subFlowRef": "uniqueSubFlowId"
+}
+```
+
+If you need to define the `version` and `waitForCompletion` parameters in your `subFlowRef` definition, you can define
+it with its `object` type which has the following properties:
+
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
-| waitForCompletion | If workflow execution must wait for sub-workflow to finish before continuing | boolean | yes |
-| workflowId |Sub-workflow unique id | boolean | no |
+| workflowId | Sub-workflow unique id | boolean | no |
```

Comment:
I believe workflowId is a required attribute so it should be set to yes in the last column

Reply:
```suggestion
| workflowId | Sub-workflow unique id | boolean | yes |
```

Reply:
@tsurdilo has this been applied?